tests/pytests/unit/grains/test_metadata_gce.py

```diff
@@ -0,0 +1,101 @@
+"""
+    Unit test for salt.grains.metadata_gce
+
+
+    :codeauthor: :email" `Thomas Phipps <tphipps@vmware.com>
+
+"""
+
+import logging
+
+import pytest
+import salt.grains.metadata_gce as metadata
+import salt.utils.http as http
+from tests.support.mock import create_autospec, patch
+
+# from Exception import Exception, ValueError
+
+log = logging.getLogger(__name__)
+
+
+metadata_vals = {
```

Comment:
Can you make this a fixture instead of a global variable?

Reply:
I'll do one better and move the global into the side_effect function it was meant for then move the function into the test function it is used in. Since can't call a fixture as a side_effect. and the function is only used in one test.